kitsune/wiki/forms.py

```diff
@@ -425,6 +425,6 @@ class RevisionFilterForm(forms.Form):
     """Form to filter a list of revisions."""
 
     locale = forms.ChoiceField(label=_lazy("Locale:"), choices=languages, required=False)
-    users = MultiUsernameField(label=_lazy("Users:"), required=False)
+    users = forms.CharField(label=_lazy("Users:"), required=False)
```

Comment:
Why are you changing this?

Reply:
I think because of the tiny input box, I didn't imagine anyone actually trying to input more than one name into the filter. And the lack of directions about it (like 'You can enter more than one name, separated by a comma.'), made it seem even less likely. 

That said, I probably overstepped, and will revise....